vtr_flow/arch/no_timing/memory_sweep/k4_N10_memSize16384_memData64.xml

```diff
@@ -579,7 +579,7 @@
       <output name="out" num_pins="72"/>
       <mode name="two_divisible_mult_18x18">
         <pb_type name="divisible_mult_18x18" num_pb="2">
-          <input name="a" num_pins="18"/>
+          <input name="aa" num_pins="18"/>
```

Comment:
Not sure why the artifacts requires some changes in the XML files.

Reply:
This is the artificial bug created to fail one of the tests so that we can check artifact uploading. I will revert it before merging to master